add-on/panels/js/roomViews.jsx

```diff
@@ -521,6 +521,9 @@ loop.roomViews = (function(mozL10n) {
         var roomTitle = this.state.roomName ||
                         this.state.roomContextUrls[0].description ||
                         this.state.roomContextUrls[0].location;
+        if (typeof roomTitle === "object" && roomTitle === null) {
+          roomTitle = " ";
```

Comment:
I think we can change the if statement here to just check `!roomTitle`. There's no need to be more specific.

Also, please can you change what it sets the replacement title to:

    roomTitle = mozL10n.get("room_name_untitled_page");

That should display "Untitled Page" like the room list in the panel. I think that'll be slightly more consistent.